.github/workflows/test-rust.yaml

```
@@ -54,6 +54,19 @@ jobs:
         with:
           command: test
           args: ${{ inputs.target_option }} --no-run --locked
+      - name: Run docker compose
+        run: docker compose up -d
+        working-directory: ./prql-compiler/tests/integration-rdbms
+        if: ${{ inputs.os == 'ubuntu-latest' }}
+      - name: Wait for database
+        uses: ifaxity/wait-on-action@v1
+        with:
+          resource: "tcp:5432 tcp:3306 tcp:1433"
+          timeout: 60000
+        if: ${{ inputs.os == 'ubuntu-latest' }}
+      - name: skip integration if not linux
+        run: echo "SKIP_INTEGRATION=true" >> $GITHUB_ENV
```

```suggestion
        run: echo "SKIP_INTEGRATION=true" >> "$GITHUB_ENV"
```
I think this is what the linter wants